index.html

```diff
@@ -1,22 +1,237 @@
+					</div>
+					<div class="likes">
+						<i class="far fa-thumbs-down"></i>
+						<p id="number-dislikes">138</p>
+					</div>
+					<div class="likes">
+						<i class="fas fa-share"></i>
+						<p id="share">SHARE</p>
+					</div>
+					<div class="likes">
+						<i class="fas fa-list"></i>
+						<p id="save">SAVE</p>
+					</div>
+					<div class="likes">
+						<i class="fas fa-ellipsis-h"></i>
+					</div>
+				</div>
+				<div id="description-container">
+					<div id="first-part">
+						<div id="channel">
+							<div id="user-image-container">
+								<img id="user-image"
+									src="https://yt3.ggpht.com/ytc/AKedOLT6ck5p3kvuEMSLqzf9aviTcwthTLwoSZnjuJAvhw=s176-c-k-c0x00ffffff-no-rj"
+									alt="This should be a Rick Beato image">
+							</div>
+							<div id="channel-info-container">
+								<p id="user-name">Rick Beato</p>
+								<p id="subscribers">2.59 M subscribers</p>
+							</div>
+						</div>
+						<div id="subscribe-button">
+							<button id="button">SUBSCRIBE</button>
+						</div>
+					</div>
+					<div id="video-description">
+						<p></p>
+						<p></p>
+					</div>
+				</div>
+			</section>
+			<section id="comment-section">
+				<div id="comments-first-part">
+					<p id="number-of-comments">1,746 comments</p>
+					<div id="order-comments">
+						<i class="fas fa-sort-amount-down"></i>
+						<p id="order-comments-by">ORDER BY</p>
+					</div>
+				</div>
+				<div id="comments-second-part">
+					<img id="comment-section-img"
+						src="https://yt3.ggpht.com/ytc/AKedOLT6ck5p3kvuEMSLqzf9aviTcwthTLwoSZnjuJAvhw=s176-c-k-c0x00ffffff-no-rj"
+						alt="This should be a Rick Beato image">
+					<div id="comment-input-container">
+						<input id="comment-input" type="text" placeholder="Add a public comment...">
+					</div>
+				</div>
+				<div id="comment">
+					<div class="comment-container">
+						<img id="comment-img"
+							src="https://yt3.ggpht.com/yti/APfAmoG6vx3aoJ9HXjlLcgwnzbT8xvF4f2nE4GT9IBVi=s88-c-k-c0x00ffffff-no-rj"
+							alt="This should be an user icon">
+					</div>
+					<div class="comment-info-container">
+						<div class="comment-first-part">
+							<p class="user-name">Santiago Velosa</p>
+							<p class="comment-time">A day ago</p>
+						</div>
+						<p class="actual-comment">This is a very inspiring story, thank you so much for sharing, Rick!
+						</p>
+						<div class="comment-interaction">
+							<div class="likes">
+								<i class="far fa-thumbs-up"></i>
+								<p>648</p>
+							</div>
+							<div class="comment-response">
+								<i class="far fa-thumbs-down"></i>
+								<p>REPLY</p>
+							</div>
+						</div>
+					</div>
+				</div>
+			</section>
+		</section>
+		<section id="thumbnails">
+			<div id="thumbnail-container">
+				<ul>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=O_LVl8w4-vw" target="_blank">
+								<img id="img"
```

Comment:
You cannot have more than one element with the same id in an HTML document. Please change it to class